app/src/main/res/layout/fragment_client_settings.xml

```diff
@@ -86,6 +88,28 @@
                 />
     </RadioGroup>
 
+    <SeekBar
+            android:id="@+id/noiseDetectionSeekBar"
+            android:layout_width="0dp"
+            android:layout_height="50dp"
```

Comment:
To constants

Reply:
Why? This value isn't used in any other place.

Reply:
Ok, i thought i saw that value somewhere earlier